src/core/Refiner.ml

```diff
@@ -66,10 +66,11 @@ end
 
 module Probe : sig
   val probe_chk : string option -> T.Chk.tac -> T.Chk.tac
+  val probe_chk_no_bdry : string option -> T.Chk.tac -> T.Chk.tac
   val probe_syn : string option -> T.Syn.tac -> T.Syn.tac
 end =
 struct
-  let print_state lbl tp : unit m =
+  let print_state lbl tp: unit m =
```

Comment:
I believe the surrounding style is to have space before the colon... @jonsterling